controllers/platform/services/services.go

```diff
@@ -114,8 +114,8 @@ func (d DataIndex) GetEnvironmentVariables() []corev1.EnvVar {
 func (d DataIndex) GetPodResourceRequirements() corev1.ResourceRequirements {
 	return corev1.ResourceRequirements{
 		Limits: corev1.ResourceList{
-			corev1.ResourceCPU:    resource.MustParse("100m"),
-			corev1.ResourceMemory: resource.MustParse("256Mi"),
+			corev1.ResourceCPU:    resource.MustParse("500m"),
+			corev1.ResourceMemory: resource.MustParse("1Gi"),
 		},
```

Comment:
Is there any justification for this change? Have you run a benchmark? @wmedvede do you have any idea regarding DI/JS consumption resources? Can we have a follow-up task to get a more approximated number? I feel like this can be too much for a default setup. Mainly when running it locally.

Reply:
I got OOMKill with 256Mi for the DI. I increased it to 512Mi as well as the CPU limits to try to speed up the deployment as it takes 2:40 seconds for the container to reach ready status, which is significant more than for the Job Service (90 seconds). 

It didn't help much on both accounts. I can reduce it to 512Mi and the CPU to 100m for limits if you think it aligns with your expectations. 

Is there is any previous test done on the resource limits for the DI container or these values were selected on best effort?

Reply:
Don't worry about changing these numbers now, we can do a benchmark later and have close numbers and a good approximation for users depending on their env.

Reply:
Ack. I'll set the memory request and limit to 512Mi to avoid random OOMKills. I wonder if the startup time is caused by the JVM resizing its memory capacity as it runs the code....

Reply:
I can confirm that on OCP cluster the data-service failed to start with the default values set by the operator and had to set the resources limits in the platform CR explicitly to make it start:
```
  services:
    dataIndex:
      enabled: true
      podTemplate:
        container:
          image: "quay.io/kiegroup/kogito-data-index-postgresql-nightly:latest" # To be removed when stable version is released
          resources:
            limits:
              cpu: 500m
              memory: 512Mi
      persistence:
```